aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,29 +23,69 @@ import { Example } from '../../layouts';
   <SimpleSignUpExample />
 </Example>
 
-## Guidance
+### Jump to section...
+[When to use/not to use](#use)</br>
```

Comment:
`<br />` is unnecessary. In Markdown, a new line should automatically occur with Return/Enter key. If that doesn't happen, add two spaces at the end of the line, then hit enter. The two spaces tells markdown to do the break row. Usually that is not necessary.

Update the jump to link to read `#when-to-use` (see corresponding comment below regarding auto creation of heading ids)

```suggestion
[When to use/not to use](#when-to-use)
```

Reply:
Yeah, it didn't seem to be accepting the line break, so I used the html code. Someone in the wwwworld suggested that double spaces aren't as good as the <br> anyway bc there are some people who still use double spaces after a full stop. But good call.

Reply:
If you do use `br`, make sure the `/` comes after the `br`. I think it is reversed and causing parsing issues.

Reply:
strange I attempted to fix this however still not doing the line break either my markdown skills are rusty or Im tired 😴  @halocline if you can take a stab I fixed the links so the preview shows. I tried `\` `two spaces` and `<br />` and still wasn't creating the line break 😕

Reply:
I think this is a quirk unique to this particular markdown parser. 

Unfortunately, I don't think we can get the desired presentation with pure markdown. I will propose an alternative.

Reply:
okay thank you Im glad it wasnt just me!